src/SourceLocator/SourceLocator.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Asgrim\SourceLocator;
+
+interface SourceLocator
```

Comment:
These things are not actually locating a class as such, it is actually locating source code (the `locate` method should return a `LocatedSource` VO which contains just source code + filename... it is the `Reflector` which is loading that source code and finding out if the class actually exists within that source code... so IMO the name `SourceLocator` is actually more accurate (unless you can suggest something _more_ appropriate)